### ISSUE TITLE
Add different type of directions

### ISSUE THREAD

**Issue by diskmanti** [CityZenApp/CityZen-Android#10]
When users search for directions give them the possibility to request directions by foot, car, bike ,etc..

**Comment by jbelien** [comment]
Which API do you use to calculate the direction ?

**Comment by rskikuli** [comment]
@diskmanti the library we are using to calculate the directions and guide uses only. As far as we know there is no floss library that gives directions for 'on foot' and 'bike' usage. 
@jbelien I think @valdio has used this one https://github.com/MKergall/osmbonuspack. Do you know if there is any library that can be used for the direction feature? That would really help.

**Comment by jbelien** [comment]
@rskikuli Ben Abelshausen ( @xivk ), one my co - OSMBE board member, developed an open-source library just for that : https://github.com/itinero/routing
I saw it in action and it's really incredible. If you need it, I probably can put you in touch with him :)

**Comment by xivk** [comment]
I agree with @jbelien that mine is awesome! ;-) But it's only usable via Xamarin, at least not without some extra effort, so not really something you guys would be planning to I assume.

Also have a look at [graphhopper](https://github.com/graphhopper/graphhopper) by @karussell

**Comment by jbelien** [comment]
> But it's only usable via Xamarin, at least not without some extra effort, so not really something you guys would be planning to I assume.

Oh damn ... That's too bad :(

Indeed GraphHopper seems to have an Android version : https://github.com/graphhopper/graphhopper/tree/master/android

**Comment by diskmanti** [comment]
I also used graphhopper for some web apps and I was happy with the results. Anyway if it is too much trouble to implement this feature than better concentrate on other things. It was more of a feature request than something critical to the app.

**Comment by karussell** [comment]
@xivk Thanks a bunch for the mention :+1: much appreciate also your hard work on itinero :) !

**Comment by chilledfrogs** [comment]
Hello there,

Just wanted to know how this is coming along, since for me I don't even have a car and I would really need (ideally turn-by-turn but not a must) directions on foot as well as on a bike... In any case, great job on your app, also looking forward to a dark theme 😅
